frontend/src/sections/JoinCommunity/JoinCommunity.tsx

```diff
@@ -27,115 +27,135 @@ function Join() {
           </p>
         </div>
 
-        <div className='w-96 md:w-auto mt-10 relative'>
-          <div className='relative '>
-            <Image
-              src='/Phone2.png'
-              alt='Phone Image'
-              width={563}
-              height={707}
-              className='object-contain scale-110'
-            />
-            <div className='absolute left-[72px] rounded-[78px] z-30 w-fit h-fit md:w-[505px] md:h-[707px] inset-1'>
-              <div className='flex flex-col justify-start md:justify-center items-start md:items-center gap-6 md:gap-10 pt-20 md:pt-35 md:mt-8 ml-2 md:ml-0'>
-                <div className='flex flex-row justify-center items-center gap-7 md:gap-10'>
-                  <Link
-                    href='https://www.instagram.com/djangoindia'
-                    target='_blank'
-                    passHref
-                  >
-                    <Image
-                      src='/icons/insta.svg'
-                      alt='Instagram'
-                      width={width > 768 ? 72: 50}
-                      height={width > 768 ? 101 : 50.5}
-                      className='transition transform hover:scale-110'
-                    />
-                  </Link>
+        {/* <div className='md:w-auto mt-10 relative'> */}
+        <div className='relative '>
+          <Image
```

Comment:
Maybe we should remove the phone image and keep the social media handles only. The reason i am suggesting this because on mobile the UI looks a little bit shifted to right because of dropshadow in the image.
@DevilsAutumn 

![image](https://github.com/user-attachments/assets/e4be783e-0801-4289-95a1-8fd78e389eee)

Lets keep the phone image only when there is some content on the left side of it and hide on small screens.

Reply:
Can't remove the phone image its one of the attractive component of our website :)

Reply:
Can think of some other image here then, but the UI should be consistent on ever viewport. Also i was  suggesting to hide it on small viewport only and not others